File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int i =0;
        int j = 0;
        int n = nums.length;
        while(j < n){
            nums[i] = nums[j];
            while(j < n && nums[i] == nums[j]){
                j++;

            }
            i++;
        }
        return i;
        
    }
}

File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        int ch;
        int n = nums.size();
        for(int i=0;i<n;i++){
            ch = nums[i];
            for(int j=i+1;j<n;j++){
                if(ch==nums[j]){
                     return true;
                }
            }
        }
        return false;
    }
};